CSharp/JobPrep/Program.cs

```diff
@@ -0,0 +1,333 @@
+﻿#define VERBOSE
```

Comment:
Why bother using the preprocessor for this...?  Just define a static Boolean variable in the class and use that instead...?  Possibly set it via CmdLine parameter?  Using #define is weird.